src/js/messaging/actions/folders.js

```diff
@@ -56,11 +57,21 @@ export function toggleManagedFolders() {
 }
 
 export function createNewFolder(folderName) {
-  // TODO: Actually make the request
-  // that creates the folder
-  return {
-    type: CREATE_NEW_FOLDER,
-    folderName
+  // create JSON payload
+  const folderData = {};
+  folderData.folder = folderName;
```

Comment:
As explained by department-of-veterans-affairs/vets-api#168, we should be sending:

```
{
  folder: {
    name: folderName
  }
}
```